docs/keycloak-configuration.md

```diff
@@ -26,153 +33,313 @@ To create a new realm follow the steps below:
-Under the "Role mapping" tab, map the role/s associated with the group using "Assign role"
-E.g. assign roles to the "Admin-group" account, select "tornjak-admin-realm-role"
-Press Assign
-
-Figure 30: Role Mapping to Group PageFigure 31 Created Groups PageNow one can assign these groups to users accordingly.
-Adjusting Optional Settings Under Realm Settings
+
+- Under your realm (Tornjak) > Groups
+- Click on Create group
+- Give your group a "Name" (Admin-group)
+- Click Save
+- Repeat to create "Viewer-group"
+
+![Create Group Page](/docs/rsrc/keycloak_diagrams/29-create-group.png)
+
+
+- Select the group name from the list
+- Under the "Role mapping" tab, map the role/s associated with the group using "Assign role"
+- E.g. assign roles to the "Admin-group" account, select "tornjak-admin-realm-role"
+- Press Assign
+
+![Role Mapping to Group Page](/docs/rsrc/keycloak_diagrams/30-role-mapping.png)
+
+
+![Created Groups Page](/docs/rsrc/keycloak_diagrams/31-created-groups.png)
+
+
+Now one can assign these groups to users accordingly.
+
+
+## Adjusting Optional Settings Under Realm Settings
+
 There are some useful realm settings to adjust. For the selected Realm Tornjak, Select "Realm settings" under the "Configure" sub-section on the left side of the panel:
-Go to the Login tab
-Toggle ON User registration- which allows users to register through your application
-Toggle ON Forgot password- which allows users to reset and retrieve forgotten passwords
-Toggle ON Remember me- which helps users remember forgotten passwords
-Toggle ON Edit username - which allows the Keycloak admin to edit the username already set otherwise it is read-only
-
-Figure 32: Realm Settings PageTo set Default Groups and default roles to users when they register:
-Go to the User registrationtab under "Realm settings"
-Click on Default groups
-Add your default group [In our case add "Viewer-group" group as default] this will make viewer group default for all users
-
-Figure 33: User Registration Realm Settings PageCreating And Registering Users
+
+- Go to the Login tab
+- Toggle ON User registration- which allows users to register through your application
+- Toggle ON Forgot password- which allows users to reset and retrieve forgotten passwords
+- Toggle ON Remember me- which helps users remember forgotten passwords
+- Toggle ON Edit username - which allows the Keycloak admin to edit the username already set otherwise it is read-only
+
+![Realm Settings Page](/docs/rsrc/keycloak_diagrams/32-realm-settings.png)
+
+
+To set Default Groups and default roles to users when they register:
+
+- Go to the User registrationtab under "Realm settings"
+- Click on Default groups
+- Add your default group [In our case add "Viewer-group" group as default] this will make viewer group default for all users
+
+![User Registration Realm Settings Page](/docs/rsrc/keycloak_diagrams/33-user-registration.png)
+
+
+## Creating And Registering Users
```

Comment:
one last nit! 

because users aren't imported, maybe make this a bigger heading: 

```suggestion
# Creating And Registering Users
```